Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -48,6 +48,31 @@ configuration:
   name: additional_malicious
   required: false
   type: 12
+- display: Incident Type
```

Comment:
arent you missing the `longRunning` parameter here?

Reply:
It is in line 4694
https://github.com/demisto/content/pull/13644/files#diff-f261a9104c2b46a147c9f66fbd3d1c8018f43e76a5d82cd0feb75294ab9e9056R4694